chapter02/chapter02.rpy

```diff
@@ -52,88 +52,86 @@ label chapter_02:
     I instantly turned my head toward the woman and smacked her hand that had began to roll up my sleeve.
     The woman took a step backwards, holding the hand I had hit close to her chest with a surprised look on her face.
-    I gritted my teeth and glared at her, rolling up my sleeve myself. [Pause] No one, except for me can touch my injuries.
+    I gritted my teeth and glared at her, rolling up my sleeve myself. [Pause] No one but me can touch my injuries.
     I scanned the tray for some sort of medicine bottle, grabbing a small round container with gel inside.
     Opening the container with a flick of my thumb, I placed the container back down on the tray and scooped up some of the gel with the tip of three of my fingers.
-    Nurse "Uh wait, you need to remove the bandage first..."
+    m "Uh wait, you need to remove the bandage first..."
     I glared at the woman, causing her to close her mouth. [Pause] I could see that there was some frustration forming in her eyes.
     Glancing over to my rolled up sleeve, there was indeed a thick roll of bandages and a strange belt of some sort that tightly secured them over my wound.
-    Using my pinky and thumb that were not coated in the strange gel, I somehow managed to remove the belt and bandages then applied the "medicine".
+    Using my pinky and thumb that were not coated in the strange gel, I somehow managed to remove the belt and bandages then applied the medicine.
     I gritted my teeth at the pain that spread through my body as I gently rubbed the gel.
-    Nurse "You are rather stubborn, aren't you?"
+    m "You are rather stubborn, aren't you?"
     I snorted in response, rubbing my fingers against the cloth that covered my legs and began to put on the bandages and strange belt.
     The woman walked forward to me and a placed the tray of food on top of my lap, causing me to tense up at the sight of it.
-    Nurse "Well, you were asleep for a while so you missed dinner. [pause] Go ahead and eat it."
+    m "Well, you were asleep for a while so you missed dinner. [Pause] Go ahead and eat it."
     The tray consisted of a large bowl of steaming rice, a plate decked with some sort of glistening meat and vegetables, and a cup with some sort of warm white liquid in it.
     I shivered at the sight of the food, all my memories flooding into me at once.
     I picked up the tray with my trembling right hand and threw it past the woman, causing a loud clattering sound to echo inside the room.
     The woman gasped at my actions, turning her attention to the mess near the door, where the tray had landed.
     After a minute or two of silence from the woman, she turned around and glared at me with frustrated and angry eyes. 
     I glared back, taking the action as some sort of threatening position.
-    Nurse "What in the name of Ion makes you think that you have the authority to just throw a perfectly well made meal to the wall like that?!"
```

Comment:
I'm okay with changing the "Nurse" to "Meadow" to keep the color and portrait, but at the same time I'm not so sure. There is an option later on that will effect the ending the player will get where the player (as Kenta) will decide whether or not they trust Meadow. By saying yes, Meadow reintroduces herself and from the point on in Kenta's perspective she will be known as Meadow. If no, she will remain as "Nurse" to him. If it's possible to change Nurse to Meadow while having the name be _shown_ as Nurse, then it's okay. I've only worked with ren'py for a little bit and I know the basics. I'm not sure if this is possible, but you've had more experience than me so maybe there's a way.